libs/events/src/types/orders.ts

```diff
@@ -27,16 +26,21 @@ export type OnPostedOrderPayload = {
   outputAmount: bigint
   inputToken: TokenInfo
   outputToken: TokenInfo
-  isEthFlow?: boolean
   receiver?: string
 }
+
+export type OnPostedOrderPayload = BaseOrderPayload & {
+  orderCreationHash?: string
+  isEthFlow?: boolean
+}
+
 export type OnFulfilledOrderPayload = {
   chainId: SupportedChainId
   order: EnrichedOrder
 }
 
-export type OnRejectedOrderPayload = OrderUidInChain & {
-  reason: string
-  errorCode?: number
-  // TODO: Potentially add all order info here, but lets keep it minimal for now
```

Comment:
Isn't the reject order part for when user rejects signature in the wallet?

Reply:
No, this type wasn't in use actually